app/models/ncr/organization.rb

```diff
@@ -23,5 +23,9 @@ def ool?
     def whsc?
       code == WHSC_CODE
     end
+
+    def ba_6x_tier1_team?
```

Comment:
maybe we should have a unit test for this and then just test one case in `spec/services/ncr/approval_manager_spec.rb` ?